backend/src/main/java/ca/bc/gov/app/service/client/ClientLegacyService.java

```diff
@@ -307,6 +308,55 @@ public Flux<ForestClientDto> searchContact(ContactSearchDto dto) {
                     client.clientNumber())
             );
   }
+  
+  @SuppressWarnings("unchecked")
```

Comment:
Remove the `@SuppressWarnings` here as this can hide possible issues. If you had to use this annotation it means that some unchecked exception is being thrown. Make sure to handle it instead of allowing it to be thrown

Reply:
Sure, I will work on this